account_invoice_subscription_per_contact/models/account_move.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2025 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        res = super().create(vals_list)
+        invoice_create_subtype = self.env.ref(
+            "account_invoice_subscription_per_contact.mt_invoice_new"
+        )
+        for item in res:
+            partner = item.partner_id.commercial_partner_id
+            partner_ids = partner.message_follower_ids.filtered(
+                lambda x: invoice_create_subtype in x.subtype_ids
+            ).mapped("partner_id")
```

Comment:
This should apply to the commercial partner, but also you may subscribe directly to the children.

```suggestion
            partners = item.partner_id | item.partner_id.commercial_partner_id
            partner_ids = partners.message_follower_ids.filtered(
                lambda x: invoice_create_subtype in x.subtype_ids
            ).partner_id
```

Anyway, doing this in `create` is not valid, as it's not performing the expected notifications, as it happens for example with created tasks. You can't use the `parent_id` mechanism in the `mail.message.subtype`, as it's already used by sales team, and doesn't cover the duality of the commercial partner/real partner, but dig inside and override one of the `mail.thread` methods for returning the partners as followers and with the proper subtypes for being notified.

Reply:
Ok, changed to another approach.